doc/dev/design/host-management.rst

```diff
@@ -0,0 +1,172 @@
+where the host status would be ONLINE, FAULT - with the Admin able to quick click a link to open the
+server's BMC interface to progress the fault.
+
+The action button is extended to cover the following host tasks
+
+* Edit *(change labels, BMC information)*
+* Enter Maintenance *(host must be online)*
+* Exit Maintenance *(host must be in maintenance mode)*
+* Reboot *(host must be in maintenance mode)*
+* Export *(export the server information as a CSV to support local configuration management processes)*
+* Drain 
+* Remove *(available for hosts in drained state)*
+
+When a server is selected, the host properties component is shown. This is made up of a tabbed
+interface; daemons, disks, osds, server information and performance details. The default tab
+selected is daemons, since this is the most common interaction point with a host.
+
+.. image:: mockups/hostdaemons.png
+
+The intent is that the admin is able to manage the cluster daemons directly from the UI, so the default
+display focuses on systemd information over container image id's. Image information is still available
+but not visible by default.
+
+Daemon Management
+=================
+When a daemon is selected the action button would provide the following
+
+* stop *(daemon must be running)*
+* start *(daemon must be stopped)*
+* restart *(daemon must be running)*
+* view logs *(only available to daemons in a running state)*
+
+The **view logs** option may use cephadm's **logs** command to gather the daemons logs, allowing them to be
+viewed, and potentially downloaded to the Admins machine in support of problem determination workflows and
+support escalations.
+
+Server Information
+==================
+The Pacific release provides more server metadata. A Server Information tab has been added that shows
+metadata about the host's configuration. 
+
+.. image:: mockups/hostinformation.png
+
+
+This data is gathered by cephadm's gather-facts command and includes *load* metrics for CPU and RAM, to
+allow the UI to provide a quick appreciation for load without relying on Prometheus.
+
+Adding a Host
+=============
+The process of adding a host to the cluster has changed to support a 'bulk' add option which utilises host
+mask syntax, and also includes pre-flight checks performed in the UI to filter out common misconfiguration
+issues.
+The mockup below shows the result of trying to add 3 hosts to the cluster, and illustrates;
+
+* pre-flight check results
+* the ability to edit host entries before attempting to add to the cluster
+* a holistic view of the hosts prior to adding to the cluster, allowing the admin to visually inspect
+  the desired hosts before selecting "Add to Cluster"
+
+.. image:: mockups/hostadd.png
+
+You'll also note that within the action menu, there is an item called **Explain**. This option would only
+be visibile when pre-flight checks fail for the host, and would show a modal dialog to explain the reason
+for the failure, when selected.
```

Comment:
I would suggest using the same style of table as we have in previous examples.
The actions should be grouped in the `New Host(s)` button or beside it in a new button, and only available when the row is selected.

Reply:
ACK - text and mockup image updated